include/IECoreScene/Primitive.h

```diff
@@ -63,6 +65,24 @@ class IECORESCENE_API Primitive : public VisibleRenderable
 		/// Variables a stored as a public map for easy manipulation.
 		PrimitiveVariableMap variables;
 
+		/// To obtain an IndexedView onto a particular primitive variable.
+		/// An IndexedView allows access to both indexed & non indexed primitive variables using a common API.
+		/// If the required interpolation & datatype do not match then a nullptr unique_ptr is returned unless throwIfInvalid is true
+		/// The returned IndexedView lifetime must be bound by the primitive variable data it is viewing.
+		template<typename T>
+		std::unique_ptr<PrimitiveVariable::IndexedView<T> > variableIndexedView(
```

Comment:
IndexedView is read-only, so I don't think we have any use for this non-const overload - the const one below should be sufficient.

Reply:
This comment didn't seem to get attached to my review for some reason, so I'm just commenting to make sure you see it...